sync_check/config.go

```diff
@@ -54,31 +44,43 @@ type Config struct {
 
 	LogLevel string `toml:"log-level" json:"log-level"`
 
-	TimeField string `toml:"time-field" json:"time-field"`
+	SourceDBCfg util.DBConfig `toml:"source-db" json:"source-db"`
 
-	BeginTime string `toml:"begin-time" json:"begin-time"`
+	TargetDBCfg util.DBConfig `toml:"target-db" json:"target-db"`
 
-	EndTime string `toml:"end-time" json:"end-time"`
+	ChunkSize int `toml:"chunk-size" json:"chunk-size"`
 
-	SplitField string `toml:"split-field" json:"split-field"`
+	Sample int `toml:"sample" json:"sample"`
 
-	SourceDBCfg util.DBConfig `toml:"source-db" json:"source-db"`
+	CheckThCount int `toml:"check-thcount" json:"check-thcount"`
 
-	TargetDBCfg util.DBConfig `toml:"target-db" json:"target-db"`
+	UseRowID bool `toml:"use-rowid" json:"use-rowid"`
 
-	ConfigFile string
+	FixSqlFile string `toml:"fix-sql-file" json:"fix-sql-file"`
```

Comment:
rename to OutputDifferentData？

Reply:
this file save sql, not different data, I think FixSqlFile is better